tools/plugins/com.liferay.ide.portlet.core/src/com/liferay/ide/portlet/core/spring/NewSpringPortletClassDataModelProvider.java

```diff
@@ -0,0 +1,186 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+
+package com.liferay.ide.portlet.core.spring;
+
+import com.liferay.ide.core.ILiferayConstants;
+import com.liferay.ide.core.util.CoreUtil;
+import com.liferay.ide.portlet.core.PortletCore;
+import com.liferay.ide.portlet.core.dd.PortletDescriptorHelper;
+import com.liferay.ide.portlet.core.operation.INewPortletClassDataModelProperties;
+import com.liferay.ide.portlet.core.operation.NewPortletClassDataModelProvider;
+import com.liferay.ide.sdk.core.SDK;
+import com.liferay.ide.sdk.core.SDKUtil;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.jst.j2ee.common.CommonFactory;
+import org.eclipse.jst.j2ee.common.ParamValue;
+import org.eclipse.osgi.util.NLS;
+import org.osgi.framework.Version;
+
+/**
+ * @author Terry Jia
+ */
+@SuppressWarnings( { "restriction", "rawtypes", "unchecked" } )
+public class NewSpringPortletClassDataModelProvider extends NewPortletClassDataModelProvider
+    implements INewSpringPortletClassDataModelProperties
+{
+
+    public NewSpringPortletClassDataModelProvider( boolean fragment )
+    {
+        super( fragment );
+    }
+
+    @Override
+    public boolean isPropertyEnabled( String propertyName )
+    {
+        if( INewPortletClassDataModelProperties.CREATE_JSPS.equals( propertyName ) )
+        {
+            return false;
```

Comment:
Why are these disabled?  They should be configurable just like they are in the normal portlet wizard.